include/nntile/starpu.hh

```diff
@@ -22,9 +22,8 @@
 #include <nntile/starpu/accumulate_hypot.hh>
 #include <nntile/starpu/accumulate_maxsumexp.hh>
 #include <nntile/starpu/axpy.hh>
-#include <nntile/starpu/add_slice_inplace.hh>
 #include <nntile/starpu/add_slice.hh>
-#include <nntile/starpu/add_fiber_inplace.hh>
```

Comment:
And again many headers are deleted. Seems like you started with some old version of NNTile.